pkg/events/events.go

```diff
@@ -124,7 +124,7 @@ func New(object interface{}, eventType config.EventType, resource, clusterName s
 		event.Action = eventObj.Action
 		event.TimeStamp = eventObj.LastTimestamp.Time
 		// Compatible with events.k8s.io/v1
-		if eventObj.LastTimestamp.IsZero() {
+		if eventObj.LastTimestamp.IsZero() && eventObj.Series != nil {
```

Comment:
Now I can see that I fixed the same bug as was observed by the author of PR #577. I wanted to revert that change to allow #577 to be merged, but unfortunately #577 solution is invalid and doesn't fix that problem.